backend/apps/kloudust/lib/cmd/scripts/powerOpVM.sh

```diff
@@ -3,17 +3,50 @@
 # Params
 # {1} - Domain / VM name
 # {2} - The operation to perform
+# {3} - Optional wait time before checking state (in seconds)
 
-NAME="{1}"
+VM_NAME="{1}"
 POWER_OP="{2}"
+WAIT_TIME="${3:-10}"
 
 function exitFailed() {
     echo Failed
     exit 1
 }
 
-printf "Power operating $NAME to $POWER_OP\n"
-if ! virsh $POWER_OP $NAME; then exitFailed; fi
+function backgroundCheckAndStartIfNeeded() {
+    local vm="$1"
+    local wait="$2"
```

Comment:
I don't understand why I need to pass VM name and WAIT as function params when script already has them as global constants?